src/questions/package-manager.js

```diff
@@ -0,0 +1,21 @@
+const question = {
+  type: 'list',
+  message: '📦  Choose Package Manager ',
+  name: 'packageManager',
+  choices: [
+    {
+      name: 'npm',
+      value: 'npm'
+    },
+    {
+      name: 'yarn',
+      value: 'yarn'
+    }
+  ]
+}
+
+/**
+ * return null if we can already infer the package manager
+ * from the lock file
+ */
+module.exports = projectInfos => (projectInfos.packageManager ? null : question)
```

Comment:
If you want a conditionnal render for a question, use `when` from inquirer (see an example with the prerequisite question). Also this question should not be displayed if it's not a javascript project.